staging/src/kubevirt.io/api/core/v1/schema.go

```diff
@@ -37,6 +37,8 @@ const (
 	DefaultCPUModel                        = CPUModeHostModel
 )
 
+const HotplugDiskBaseDir = "/var/run/kubevirt/hotplug-disks/"
+
```

Comment:
Naming Nit: Can you change to `HotplugDiskDir`?  I don't think we need 'base' in the name.

Reply:
Sure, will do that.

Reply:
Renamed. Thanks for the suggestion.